src/1d/FisherKolmogorov1d.cpp

```diff
@@ -0,0 +1,276 @@
+#include "FisherKolmogorov1d.hpp"
+
+void FisherKolmogorov1D::setup() {
+    namespace fs = std::filesystem;
+    try {
+        if (fs::exists(output_directory)) {
+            fs::remove_all(output_directory); // Deletes the entire directory
```

Comment:
While I understand that this is convenient, it is generally a bad idea to have a program quietly delete the whole content of a directory.